tools/nl/embeddings/input/base/2024_q3.csv

```diff
@@ -0,0 +1,8 @@
+dcid,sentence
+Count_Person_InvolvedInCrash_Motorists,Number of motorists involved in crash
+Count_Person_InvolvedInCrash_MotorVehicleOccupant,Number of motor vehicle occupants involved in crash
+Count_Vehicle_InvolvedInCrash_InTransport,Number of vehicles in motion involved in crash
+Count_VehicleCrashIncident_CollisionCrash_HeadOnCollision,Number of headon vehicle collisions
```

Comment:
a side question, is there distinction between "collision" and "crash" in schema? And shall we unify / distinct them in description?

Reply:
As per definition crash seems to be more severe with damage or fatalities, hence will stick with crash for all.